src/internal/kopia/merge_details.go

```diff
@@ -1,19 +1,76 @@
 package kopia
 
 import (
+	"github.com/alcionai/clues"
+
 	"github.com/alcionai/corso/src/internal/common"
 	"github.com/alcionai/corso/src/pkg/path"
 )
 
-type LocationPrefixMatcher struct {
+type DetailsMergeInfoer interface {
+	// Count returns the number of items that need to be merged.
+	Count() int
```

Comment:
Nit, we have lots of various counts
```suggestion
	ItemsToMerge() int
```